src/DB/projects.json

```
@@ -131,7 +131,13 @@
         "title": "temperature converter",
         "description": "This is the application for to convert the temperature from one unit to another unit.",
         "tech": ["JavaScript"]
-      }
+      },
+      {
+        "link": "https://github.com/Parmesh119/Chat_App",
+        "title": "iChat_app",
+        "description": "This is the encrypted chat_application that is only accessible to those only who have writes to access it. No other can add or edit it.",
+        "tech": ["React.js", "JavaScript"]
+      },
```

remove `,`
```suggestion
      }
```

also, make sure that you have added contributing guide, code of conduct in your repo